src/vs/workbench/parts/extensions/browser/extensionsActions.ts

```diff
@@ -394,6 +395,48 @@ export class ManageExtensionAction extends Action {
 	}
 }
 
+export class AddToWorkspaceRecommendationsAction extends Action implements IExtensionAction {
```

Comment:
Since there is an action to add an extension to recommended list, there should also be a counter action to remove it from recommended list